From: intrigeri <intrigeri@boum.org>
Date: Sat, 6 Oct 2018 05:49:58 +0000
Subject: AppArmor: allow Web Content processes to read profile.ini and the
 updates' "updater" file.

Otherwise restarting after updating Tor Browser is broken.

(cherry picked from commit 3cd8aeb0f843d2530382a61a263489aea066b15a)
---
 apparmor/torbrowser.Browser.plugin-container | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/apparmor/torbrowser.Browser.plugin-container b/apparmor/torbrowser.Browser.plugin-container
index a70f4ac..fdf5fda 100644
--- a/apparmor/torbrowser.Browser.plugin-container
+++ b/apparmor/torbrowser.Browser.plugin-container
@@ -57,7 +57,9 @@ profile torbrowser_plugin_container {
   owner @{torbrowser_home_dir}/fonts/** r,
   owner @{torbrowser_home_dir}/omni.ja r,
   owner @{torbrowser_home_dir}/TorBrowser/Data/Browser/profile.default/extensions/*.xpi r,
+  owner @{torbrowser_home_dir}/TorBrowser/Data/Browser/profiles.ini r,
   owner @{torbrowser_home_dir}/TorBrowser/UpdateInfo/updates/[0-9]*/update.{status,version} r,
+  owner @{torbrowser_home_dir}/TorBrowser/UpdateInfo/updates/[0-9]/updater rw,
   owner @{torbrowser_home_dir}/TorBrowser/Data/Browser/profile.default/startupCache/* r,
   owner @{torbrowser_home_dir}/TorBrowser/Data/Browser/profile.default/tmp/* rw,
   owner @{torbrowser_home_dir}/TorBrowser/Data/fontconfig/fonts.conf r,
